utility/src/test/java/org/oppia/android/util/networking/NetworkConnectionUtilProdImplTest.kt

```diff
@@ -148,6 +149,45 @@ class NetworkConnectionUtilProdImplTest {
     assertThat(networkConnectionUtil.getCurrentConnectionStatus()).isEqualTo(NONE)
   }
 
+  @Test
+  fun testLocalConnectionName_activeWifiConnection_returnsWifi() {
+    networkConnectionTestUtil.setNetworkInfo(
+      status = ConnectivityManager.TYPE_WIFI,
+      networkState = NetworkInfo.State.CONNECTED
+    )
+    assertThat(context.resources.getString(LOCAL.connectionName)).isEqualTo(
+      context.resources.getString(
+        R.string.network_connection_local
+      )
+    )
+  }
```

Comment:
I didn't fully understand the part `Please also add tests to verify the exact expected string is logged in the expected circumstances.`

So, as of now, I had  written test to verify that the correct string res is present for correct enum constant.

Reply:
I think that part will make more sense with my new comment in the earlier conversation thread--these should just be actual strings that are verified in tests.

Reply:
Fixed